lib/chat_api/conversations.ex

```diff
@@ -23,6 +23,19 @@ defmodule ChatApi.Conversations do
     |> Repo.all()
   end
 
+  @spec list_conversations_by_account_paginated(binary(), map()) :: Paginator.Page.t()
+  def list_conversations_by_account_paginated(account_id, filters \\ %{}) do
+    Conversation
+    |> where(account_id: ^account_id)
+    |> where(^filter_where(filters))
+    |> where([c], is_nil(c.archived_at))
+    |> order_by(desc: :last_activity_at, desc: :id)
+    |> preload([:customer, messages: [:attachments, :customer, user: :profile]])
+    |> Repo.paginate_with_cursor(
+      filter_pagination([cursor_fields: [last_activity_at: :desc, id: :desc]], filters)
```

Comment:
instead of lumping the pagination options in with `filters`, can we put it in a 3rd argument for now? then we can probably just remove the `filter_pagination` method right?

Reply:
Sounds good :+1: I moved it to the third arg